includes/settings/tools.php

```diff
@@ -212,6 +215,11 @@ private function get_rollback_versions() {
 				return [];
 			}
 
+			$plugin_information->versions = apply_filters(
```

Comment:
If there something returns from the filter there is no need to call WordPress API up there `$plugin_information = plugins_api(`

Reply:
done